sdk/videoanalyzer/video-analyzer-edge/samples/javascript/README.md

```diff
@@ -0,0 +1,159 @@
+# Azure Video Analyzer Edge client library for JavaScript
```

Comment:
The samples readme shouldn't just be a clone of the package readme. It should talk about the specific samples provided